actix-http/CHANGES.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 3.11.3
+
```

Comment:
Please drop the release version, it may cause a conflict and it would be a major change anyway 😓 

```suggestion
```